### PR TITLE
Zack Branch 1

### PR DESCRIPTION
**Pull request by Zackkogan**
added location parameter for rings #43 

requesting review from others to test additions to the code

### CODE REVIEW

File: StudyRingMeteor/collections/rings/rings.js [hunk 1]
```diff
@@ -85,6 +85,19 @@ RingSchema = new SimpleSchema({
...
+         type: String,
+         label: "location",
+        autoValue: function() {
+            //This makes sure to only set a value when it is an insert function, not an update
+            if (this.isInsert && (!this.isSet || this.value.length === 0)) {
+                return new Array();
+            }
+        },
+        autoform: {
+            type: "hidden"
```
Comment (kpamela) [inline StudyRingMeteor/collections/rings/rings.js:99]: Do not make it hidden or else, event location will not be possible to set during its creation

File: StudyRingMeteor/collections/rings/rings.js [hunk 2]
```diff
@@ -85,6 +85,19 @@ RingSchema = new SimpleSchema({
         autoform: {
             type: "hidden"
         }
+    },
+    location: {
+         type: String,
+         label: "location",
+        autoValue: function() {
+            //This makes sure to only set a value when it is an insert function, not an update
+            if (this.isInsert && (!this.isSet || this.value.length === 0)) {
```
Comment (kpamela) [inline StudyRingMeteor/collections/rings/rings.js:94]: Not necessary, unless there is a future implementation of multiple locations

**Review by kpamela — CHANGES_REQUESTED** [review]
Location was added in rings.js, move to the events collection please.

**Comment by Lordofsnails** [comment]
Brah....your code is buggy, you are missing the whole front end stuff, so basically you added a field in the collection, but no where did you request a user to input the actual location...In other words location isn t even set, hence i am failing this pull request since it does not fulfill the actual criteria

**Comment by kpamela** [comment]
@Lordofsnails usually quickForm takes care of user input, it's just in the wrong collection

**Comment by BGoldwax** [comment]
Failed Travis CI tests

**Pull request closed without merging**